runners/katacoda/templates/intro.md

```diff
@@ -0,0 +1,7 @@
+<%= description; %>
+
+<% if(tutorialPath){ %>
+The definition of each step of this tutorial can be found at https://github.com/devonfw-tutorials/katacoda-scenarios/tree/master/<%= tutorialPath; %>
```

Comment:
Is this automatically rendered to be a hyperlink?

Reply:
Yes, it is.